public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
+    <title>RDS - Learn Site</title>
+  </head>
+  <body>
+    <noscript>
+      <strong
+        >We're sorry but `RDS - Learn Site doesn't work properly without
+        JavaScript enabled. Please enable it to continue.</strong
+      >
```

Comment:
NIT: 
Can we fix the syntax of the tags? 
```
<strong>
  We're sorry but `RDS - Learn Site doesn't work properly without JavaScript enabled. Please 
  enable it to continue.
</strong>
```

Reply:
Must have missed this one. Fixed.